<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QForm_prom_create</class>
 <widget class="QWidget" name="QForm_prom_create">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1028</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1028</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>انشاء برنامج عضويه</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2"/>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1001</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(198, 207, 199);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>92</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="Qcombo_company"/>
      </item>
      <item>
       <widget class="QComboBox" name="Qcombo_branch"/>
      </item>
      <item>
       <widget class="QComboBox" name="Qcombo_cust_group"/>
      </item>
      <item>
       <widget class="QComboBox" name="Qcombo_sponsor"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>10</y>
       <width>76</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>الشركة</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>الفرع</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>مجموعة العملاء</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>نوع العضويه</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>112</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>اسم البرنامج</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>112</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>وصف البرنامج</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>فترة البرنامج</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>التاريخ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="Qdate_from">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>d/M/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>26</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>من</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>26</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>الى</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>التاريخ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="Qdate_to">
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>d/M/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>112</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="Qbtn_exit_2">
        <property name="text">
         <string>تحميل البيانات</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbtn_save">
        <property name="text">
         <string>إنشاء البرنامج</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbtn_save_4">
        <property name="text">
         <string>تعديل البرنامج</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbtn_exit">
        <property name="text">
         <string>خروج</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>الحالة</string>
    </property>
    <widget class="QRadioButton" name="Qradio_active">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>فعال</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Qradio_inactive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>موقوف</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>211</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>القيمه المستحقه</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>عدد النقاط</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>يطبق البرنامج على </string>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>BMC</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Qcombo_sponsor_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>179</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>256</x>
       <y>46</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>الباركود</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>46</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Qbtn_save_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>إضافه</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Qbtn_save_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>حذف</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>991</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="1" column="0">
     <widget class="QTableWidget" name="Qtable_promotion">
      <column>
       <property name="text">
        <string>الباركود /BMC</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>الوصف</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>الحاله</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
